core/src/main/java/org/apache/accumulo/core/conf/ClientProperty.java

```diff
@@ -37,55 +37,64 @@
 public enum ClientProperty {
 
   // Instance
-  INSTANCE_NAME("instance.name", "", "Name of Accumulo instance to connect to", "", true),
-  INSTANCE_ZOOKEEPERS("instance.zookeepers", "localhost:2181",
-      "Zookeeper connection information for Accumulo instance", "", true),
-  INSTANCE_ZOOKEEPERS_TIMEOUT("instance.zookeepers.timeout", "30s", "Zookeeper session timeout"),
+  INSTANCE_NAME("instance.name", "", PropertyType.STRING,
+      "Name of Accumulo instance to " + "connect to", "2.0.0", true),
+  INSTANCE_ZOOKEEPERS("instance.zookeepers", "localhost:2181", PropertyType.HOSTLIST,
+      "Zookeeper connection information for Accumulo instance", "2.0.0", true),
+  INSTANCE_ZOOKEEPERS_TIMEOUT("instance.zookeepers.timeout", "30s", PropertyType.TIMEDURATION,
+      "Zookeeper session timeout", "2.0.0", false),
 
   // Authentication
-  AUTH_TYPE("auth.type", "password",
-      "Authentication method (i.e password, kerberos, PasswordToken, KerberosToken, etc)", "",
+  AUTH_TYPE("auth.type", "password", PropertyType.STRING,
+      "Authentication method (i.e password, kerberos, PasswordToken, KerberosToken, etc)", "2.0.0",
       true),
-  AUTH_PRINCIPAL("auth.principal", "",
-      "Accumulo principal/username for chosen authentication method", "", true),
-  AUTH_TOKEN("auth.token", "", "Authentication token (ex. mypassword, /path/to/keytab)", "", true),
+  AUTH_PRINCIPAL("auth.principal", "", PropertyType.STRING,
+      "Accumulo principal/username for chosen authentication method", "2.0.0", true),
+  AUTH_TOKEN("auth.token", "", PropertyType.STRING,
+      "Authentication token (ex. mypassword, /path/to/keytab)", "2.0.0", true),
 
   // BatchWriter
-  BATCH_WRITER_MAX_MEMORY_BYTES("batch.writer.max.memory.bytes", "52428800",
-      "Max memory (in bytes) to batch before writing"),
-  BATCH_WRITER_MAX_LATENCY_SEC("batch.writer.max.latency.sec", "120",
-      "Max amount of time (in seconds) to hold data in memory before flushing it"),
-  BATCH_WRITER_MAX_TIMEOUT_SEC("batch.writer.max.timeout.sec", "0",
+  BATCH_WRITER_MAX_MEMORY_BYTES("batch.writer.memory.max", "50M", PropertyType.BYTES,
```

Comment:
Since names changed need to ensure the props file that ends up in the tarball is updated.

Reply:
I believe [ClientConfigGenerate](https://github.com/apache/accumulo/blob/master/core/src/main/java/org/apache/accumulo/core/conf/ClientConfigGenerate.java) will handle that.  But I will make sure I update the website.